examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p>This list should contain only the variables that we see in <code>trace.sample_stats</code>.  Therefore it will contain most varibles explained in the ArviZ schema, but not all, <code>step_size_nom</code>  or <code>int_time</code>  are not present in pymc3 but are defined in arviz because other PPLs like Stan or Turing (in Julia) do have them. It will also contain variables like <code>step_size_bar</code>  or <code>perf_counter_diff</code>  which are present here but not on ArviZ schema. Right now there are several variables which are currently replicated because they use both the internal pymc3 name and the arviz one. This <a href="in Julia" target="_blank">comment</a> might help a bit, but the important thing is to have a bullet point per variable in <code>trace.sample_stats</code>.  Nothing more but nothing less either.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
that comment is not available, i probably do not have access to that link. Anyway I tried to make the changes after having a look at NUTS stats

Reply:
Sorry,  link got corrupted when pasting,  here it is again https://github.com/arviz-devs/arviz/pull/1063#issuecomment-589771200 (I still have not checked reviewnb, currently on mobile at the train)

Reply:
@OriolAbril no problem at all! also I just checked and the items ive added are the exact same, I went through the code for sample_stats_to_array [here ](https://github.com/arviz-devs/arviz/blob/main/arviz/data/io_pymc3.py#L323 ), that clarified it for me. thanks anyway this looks like a helpful comment.

this is from the current commit on the notebook, in case you wanted to see:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/55153708/113520138-b47ef780-95ae-11eb-9947-822e4ff337bc.png">